main/modes/mode_main_menu.c

```diff
@@ -467,6 +469,7 @@ void mainMenuSetUpTopMenu(bool resetPos)
     addRowToMeleeMenu(mainMenu->menu, mainMenuTools);
     addRowToMeleeMenu(mainMenu->menu, mainMenuMusic);
     addRowToMeleeMenu(mainMenu->menu, mainMenuSettings);
+    addRowToMeleeMenu(mainMenu->menu, mainMenuManual);
```

Comment:
Didn't want to consider it a tool, eh?

Reply:
Ah, I just stuck it on the main menu because there was room and I figured it would be nice to have up front. But tools is probably a better place for it.

Reply:
This will no longer fit on the main menu after recent changes merged into `main`. Tools seems like a better fit anyway! :)

Reply:
Moved to bottom of tools menu